pkg/configtx/policies.go

```diff
@@ -214,7 +203,8 @@ func (c *ConfigTx) RemoveOrdererPolicy(policyName string) error {
 		return err
 	}
 
-	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey], policyName, policies)
+	removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey], policyName, policies)
```

Comment:
Same comments for this function as the above, missing some comments for panicking/erroring

Reply:
This function will not panic because `OrdererPolicies()` checks for the existence of the Orderer group.